wordpress-mobile/gutenberg-mobile
# Inserter - block icons

This PR makes the block icons visible in the `BlockPicker`:

<img width="262" alt="screen shot 2018-08-23 at 21 48 42" src="https://user-images.githubusercontent.com/6597771/44559870-30d8ab00-a723-11e8-86a5-8c9569a6d95c.png">

To test:
1. update the branch
2. focus on a block and tap the `+` icon
3. observe the picker now shows the block icons

Relies on the following Gutenberg PR https://github.com/WordPress/gutenberg/pull/9294
